owtf/api/handlers/test_group.py

```diff
@@ -0,0 +1,141 @@
+"""
+owtf.api.handlers.test_group
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+
+
+from owtf.api.handlers.base import APIRequestHandler
+from owtf.models.plugin import Plugin
+from owtf.models.test_group import TestGroup
+from owtf.lib.exceptions import APIError
+from owtf.managers.plugin import get_plugins_by_group
+
+
+__all__ = ["GroupDataHandler"]
+
+
+class GroupDataHandler(APIRequestHandler):
+    """Get completed plugin output data from the DB."""
+
+    SUPPORTED_METHODS = ["POST"]
+
+    
+    def post(self, action):
+        """Create a new plugin group.
+
+        **Example request**:
+
+        .. sourcecode:: http
+
+            POST /api/v1/group/add HTTP/1.1
+            Accept: application/json
+            accept-language: en-US,en;q=0.9,
+            cache-control: no-cache,
+            content-type: application/x-www-form-urlencoded;charset=UTF-8,
+            pragma: no-cache
+
+            plugin_data: group=network%3Btest&type=active&code=PTES-001&id=3
+
+
+         **Example response**:
+
+        .. sourcecode:: http
+
+            HTTP/1.1 200 OK
+            Content-Encoding: gzip
+            Vary: Accept-Encoding
+            Content-Type: application/json
+
+
+            {
+                "status": "success", 
+                "data": "net;test"
+            }
+
+
+        """     
+        plugin_data = self.get_argument("plugin_data", default=None)
+        if action == "add":
+            plugin_data_list = plugin_data.split("&")
+            plugin_dict = {}
+            for plugin_element in plugin_data_list:
```

Comment:
this parsing seems weird -- I think urlparsing these parameters into a dictionary or a list would be a cleaner approach.